ARCHITECTURE.md

```diff
@@ -16,11 +16,13 @@ There are 3 folders in `src`, corresponding to the 3 processes that the VHS runs
   * `/network/validator_reports`: Returns scores for the nodes that it has crawled in the last day.
   * `/network/topology`: Returns information about all the nodes that the crawler has crawled in the last hour.
   * `/network/topology/nodes`: Same as above.
-  * `/network/topology/nodes/:publicKey`: Returns information about a specific node.
+  * `/network/topology/nodes/:network`: Same as above, filtered to only return the nodes that are a part of `network`.
+  * `/network/topology/node/:publicKey`: Returns information about a specific node.
```

Comment:
For this, does the URL have the word "node" in it or a specific node's identifier? (How does someone specify which specific node should be returned?)

Reply:
Oh, is the publicKey the parameter?

Reply:
The `:` before a word indicates that it's a variable/parameter